packages/search-ui/src/actions/setSearchTerm.js

```diff
@@ -5,12 +5,13 @@
  *
  * @param searchTerm String
  * @param options Object Additional objects
+ * @param options.autocompleteResults Fetch autocomplete results?
```

Comment:
Can you please create an issue for autocomplete/debounce documentation? 👍

Reply:
Hey @goodroot. I just added a subtask here: https://github.com/elastic/search-ui/issues/105.

I don't mind updating the README / documentation to include that option, but you're certainly welcome to if you'd prefer.